src/ui/hexedit.cc

```diff
@@ -42,6 +43,8 @@ void HexEdit::recalculateValues() {
   verticalByteBorderMargin_ = charHeight_ / 5;
   dataBytesCount_ = dataModel_->binData().size();
   byteCharsCount_ = (dataModel_->binData().width() + 3) / 4;
+  // when width is 64 then still it is correct
```

Comment:
Unfortunately 2**64 is not representable as ULL, so this will be a undefined behaviour :(